src/v2/guide/comparison.md

```diff
@@ -55,55 +53,55 @@ If you're feeling lazy though, below are the numbers from one run in Chrome 52 o
   </thead>
   <tbody>
     <tr>
-      <th>Fastest</th>
+      <th>Plus rapide</th>
       <td>23ms</td>
       <td>63ms</td>
     </tr>
     <tr>
-      <th>Median</th>
+      <th>Médianne</th>
       <td>42ms</td>
       <td>81ms</td>
     </tr>
     <tr>
-      <th>Average</th>
+      <th>Moyenne</th>
       <td>51ms</td>
       <td>94ms</td>
     </tr>
     <tr>
-      <th>95th Perc.</th>
+      <th>Pour 95</th>
       <td>73ms</td>
       <td>164ms</td>
     </tr>
     <tr>
-      <th>Slowest</th>
+      <th>Plus lent</th>
       <td>343ms</td>
       <td>453ms</td>
     </tr>
   </tbody>
 </table>
 {% endraw %}
 
-#### Update Performance
+#### Performance de mise à jour
 
-In React, when a component's state changes, it triggers the re-render of the entire component sub-tree, starting at that component as root. To avoid unnecessary re-renders of child components, you need to implement `shouldComponentUpdate` everywhere and use immutable data structures. In Vue, a component's dependencies are automatically tracked during its render, so the system knows precisely which components actually need to re-render.
+Avec React, quand l'état d'un composant change, cela enclenche de nouveau le rendu de tous les sous-composants inclues, en commençant par le composant racine. Pour empêcher le re-rendu des composants enfants, vous devez implémenter `shouldComponentUpdate` partout et utilisez des structures de données immuables. Dans Vue, les dépendances d'un composant sont automatiquement suivies, ainsi le système sait précisément quels composants ont besoin d'être re-rendu.
 
-This means updates in unoptimized Vue will be much faster than unoptimized React and actually, due to the improved render performance in Vue, even fully-optimized React will usually be slower than Vue is out-of-the-box.
+Cela signifie que les mises à jour dans un Vue basique seront plus rapide que celles d'un React basique, et grâce au rendu performant de Vue, même un React pleinement optimisé est plus lent qu'un Vue basique.
```

Comment:
« dans son état initial » me fait penser « sans qu'on change la configuration interne ».
Hors « out-of-the-box » veut plutôt dire « sans qu'on y est ajouté d'autre chose ».

Aussi j'ai changé par « que Vue sans optimisations. » puisque que c'est ce que l'on compare ici.